litmus/director/TCID-DIR-OP-INSTALL-OPENEBS-DP-ON-SPECIFIC-NODE/test.yml

```diff
@@ -226,47 +226,53 @@
+            url: "{{ director_url }}/v3/groups/{{ group_id }}/nodes/{{ node_id[0] }}"                                                                                                                                   
+            method: GET                                                                                                                                                                                                 
+            url_username: "{{ username.stdout}}"                                                                                                                                                                        
+            url_password: "{{ password.stdout}}"                                                                                                                                                                        
+            force_basic_auth: yes                                                                                                                                                                                       
+            return_content: yes                                                                                                                                                                                         
+            body_format: json                                                                                                                                                                                           
+            status_code: 200                                                                                                                                                                                            
+          register: node1                                                                                                                                                                                               
+          until: "'mayadata.io/data-plane' in {{node1.json.labels | to_json | string }}"                                                                                                                                
+          delay: 2                                                                                                                                                                                                      
+          retries: 60
+
+        ## Checking the control plane label on the second node                                                                                                                                                          
+        - name: Checking the label on the second node of the cluster                                                                                                                                                    
```

Comment:
Rename the task to - `Checking control plane label on the second node of the cluster`